src/metric.rs

```diff
@@ -77,7 +79,7 @@ pub trait Metric: Monoid + Eq + Add<usize, Output=Self>
 }
 
 #[derive(Clone, Copy, Debug, PartialOrd, Ord, PartialEq, Eq)]
-pub struct Grapheme(usize);
+pub struct Grapheme(pub usize);
 impl Default for Grapheme {
```

Comment:
I think I opted to go with the `Default` trait rather than `Zero` trait because if I recall correctly, `Zero` is unstable?